enable-travis-under-nodejs.md

```diff
@@ -0,0 +1,82 @@
+# How to enable Travis in a repository under the Node.js Github organization
+
+Since we don't allow third-party access to the Node.js Github organization
+except our own bots, when a repository is transferred into the Node.js
+foundation, Travis won't be able to update the build status of Pull requests
+automatically. Instead, we need to use our own github bot to pull the status
+from Travis and send the update it Github. This is what needs to be done:
+
+### Step 1: Travis setup
+
+Make sure Travis can listen to the push and pull request events of the
+repository. Go to "Settings -> Integrations & services" page of the
+repository. If it has previously configured Travis, then Travis should be
```

Comment:
If it has -> If it has been

Reply:
`If it has been previously configured Travis` sounds a bit weird as well. Maybe `If it has previously been integrated with Travis`?

Reply:
Hmm...or `If Travis has been integrated with the repository before`